fili-core/src/test/groovy/com/yahoo/bard/webservice/druid/model/query/WeightEvaluationQuerySpec.groovy

```diff
@@ -214,7 +214,7 @@ class WeightEvaluationQuerySpec extends Specification {
 
         and: "The weight-check query doesn't have a sort at either level"
         !new WeightEvaluationQuery(groupByQuery, 1).limitSpec.columns
-        !new WeightEvaluationQuery(groupByQuery, 1).innermostQuery.limitSpec.columns
+        !new WeightEvaluationQuery(groupByQuery, 1).innermostQuery.limitSpec.schema.columns
```

Comment:
Yeah, I'm pretty sure you're right, since `limitSpec` doesn't have a schema 😉 Let me give this PR a little sprucing.